homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,48 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
+    
+    
+    int tab[sequence+1];
```

Comment:
Read about std::array<>

This is C++11 feature, and we should always use it instead raw C array.

you can use it like this: std::array<int, sequnece + 1> tab;

Also tab is not a good name. array is better, because we should always use English names.